README.md

```diff
@@ -17,10 +17,57 @@ All users open a shared link to a YouTube video and then join the room. Please n
 The video will sync automatically when the host plays/pauses the video or seeks the progress bar of the video
 
 ### When you join a room
-<a href="https://imgbb.com/"><img src="https://i.ibb.co/mc8d4fh/Screenshot-from-2022-12-13-18-05-34.png" alt="Screenshot-from-2022-12-13-18-05-34" border="0" /></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/mc8d4fh/Screenshot-from-2022-12-13-18-05-34.png" alt="Screenshot-from-2022-12-13-18-05-34" border="0" width=30% /></a>
 
 ### Chat feature
-<a href="https://imgbb.com/"><img src="https://i.ibb.co/0VP476c/Screenshot-from-2022-12-13-18-09-37.png" alt="Screenshot-from-2022-12-13-18-09-37" border="0" /></a>
+<a href="https://imgbb.com/"><img src="https://i.ibb.co/0VP476c/Screenshot-from-2022-12-13-18-09-37.png" alt="Screenshot-from-2022-12-13-18-09-37" border="0" width=30%/></a>
+
+## 🔧 How to get the extension loaded?
+### Steps : 
+1.  Download the zip of this repository (because you might face some issues if you try to get the repository on your local using command line) 
+
+![1](https://user-images.githubusercontent.com/107767172/208070710-a3680da4-17e5-4599-b12f-28f0e19be732.png)
+
+
+2. Unzip the downloaded zip file.You should be able to see these folders.
+
+![2](https://user-images.githubusercontent.com/107767172/208070773-205977ca-6336-4c39-9a55-179d9bda4c8f.png)
+
+
+3. Open chrome and click on extensions icon on top-right side of the window.
+
+![3](https://user-images.githubusercontent.com/107767172/208070790-83f393ee-72c3-4154-99a0-721fa60e1978.png)
+
+
+4. Click on "Manage extensions". 
+
+![4](https://user-images.githubusercontent.com/107767172/208070813-9c526e53-a078-4e43-a006-e49604636297.png)
+
+
+5. Click on "Load Unpacked".
+
+![5](https://user-images.githubusercontent.com/107767172/208070833-94828c46-d1e6-4ca1-a9a7-d1eefae95ee7.png)
+
+
+6. Click on "extension" folder and then on "Select Folder".
+
+![6](https://user-images.githubusercontent.com/107767172/208070860-eb5ee12f-ec63-4870-8fb2-b5e4d9c1102e.png)
+
+ 
+ ### Your extension is successfully loaded
+ ### It should look like this :  
+ ![Screenshot 2022-12-16 154116](https://user-images.githubusercontent.com/107767172/208075717-329ff658-b6a4-4664-9aba-e8de20c07cbd.png)
+
+
+### To start the server locally : 
+
+ 1. Open the "server" folder that we downloaded earlier.
+ 2. Open command line there and run this command :    **node index.js**  
```

Comment:
Add npm install before this as you need to install the packages

Reply:
I meant just add the command "npm install" so that we install all the dependencies